### PR TITLE
fix: spelling errors and capitalization 

### PR DESCRIPTION
**Pull request by linden**
Some minor spelling corrections and capitalization on WebView.

### CODE REVIEW

File: docs/en/usage/recipes/bundler/sidecar.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Here is a sample to illustrate the configuration, this is not a complete `tauri.
 This way, you may [execute commands with Rust](https://doc.rust-lang.org/std/process/struct.Command.html) in your Tauri application.
 
 <Alert title="Note">
-Tauri provides some utilitary functions to handle standard cases (like loading platform specific binaries), such as:
+Tauri provides some utilitarian functions to handle standard cases (like loading platform specific binaries), such as:
```
Comment (nklayman) [inline docs/en/usage/recipes/bundler/sidecar.md]: I think we should just drop `utilitarian` here, it sounds a little off to me.

**Review by nklayman — APPROVED** [review]
Thanks for this PR, you fixed a lot of spelling mistakes and inconsistencies!

File: docs/en/usage/recipes/bundler/sidecar.md (hunk 1 above)
Comment (linden) [reply docs/en/usage/recipes/bundler/sidecar.md]: Ya I agree.

**Comment by Laegel** [comment]
As @nklayman said, thanks for this PR, @linden ! We're going to merge it soon. :)
Just know that what you've modified in `api/rust/**` will be overwritten on the next release as Markdown docs are generated from our Rust source code in [Tauri main repo](https://github.com/tauri-apps/tauri).
I'm going to write a note about that, it's not the first time such problem happens.

**Pull request merged**